### DISCUSSION TITLE
Edit extension with only one language is broken

### DISCUSSION THREAD

**Discussion by nzyeehaa** [datenstrom/community#636] [Report a bug] [answered]
0.8.17

if you remove french and german, 
edit.js throws several console errors relating to `settings` &  `.yellow-pane size` for `#yellow-pane-account`

`edit.js:380 Uncaught TypeError: Cannot set property 'checked' of null`

`<div class="yellow-pane" id="yellow-pane-menu" style="display: none; left: 318px; top: 40px;">`
`<div class="yellow-pane" id="yellow-pane-account" style="display: block;">`

yellow-language.ini shows one language
CoreMultiLanguageMode: 0

**Comment by markseuffert** [comment, marked as answer]
Thanks for your bug report. I updated the [edit extension](https://github.com/annaesvensson/yellow-edit). When you log in with your user account and there's only one language, the language selection is not shown. The JavaScript code tried to access a DOM element in `yellow-pane-account` that didn't exist. With debug level 1, one can bypass the file bundler and get the exact position.

Should be fixed now, please try it out.

**Comment by nzyeehaa** [comment]
that cleaned it up. thanks
